lib/tasks/report_primary_organisation.rake

```diff
@@ -0,0 +1,79 @@
+namespace :report_primary_organisation  do
+  desc "A one-off CSV report of how many documents were initially published by authors who don't belong to the document's primary organisation"
+  task :generate_csv, [:user_email] => :environment do |t, args|
+    require 'csv'
+
+    class CSVReport
+      def get_authors(edition)
+        edition.publication_audit_entry.edition.authors.first
+      end
+
+      def author_belongs_to_diff_org?(author, organisation)
+        author.organisation.present? && organisation.present? && organisation.id != author.organisation.id
+      end
+
+      def header_row
+        ["An admin link",
+         "Date first published",
+         "Title",
+         "Format",
+         "Lead organisation",
+         "First Author name",
+         "First Author email address",
+         "First Author organisation",
+         "Last Author name",
+         "Last Author email address",
+         "Last Author organisation"]
+      end
+
+      def get_last_author(edition)
+        edition.latest_edition.published_by
+      end
+
+      def filtered_editions
+        Edition.where("state = 'published' AND first_published_at >= '2014-01-01'")
```

Comment:
There is, it's just "magic" and not defined directly on the class.  We get it from [the transitions gem](https://github.com/troessner/transitions#automatic-scope-generation) which we used to define the state machine in `Edition::Workflow`.